### PR TITLE
Feature/release prep

### PR DESCRIPTION
**Pull request by mmiesch**
## Description

This has various changes done in preparation for the release.

The main change is having two singularity recipe files for generating the tutorial container.  To generate the tutorial container from a public release, no ssh authentication is needed: the code is public.  But, in the weeks before the release there is always a testing phase when we have to access the release branches of the internal repos.  So, I added a new file called `Singularity.tutorial-test`.    I also made it a best practice to clone the individual repos with a depth of 1 to minimize the size of the container.

### Issue(s) addressed

None

## Acceptance Criteria (Definition of Done)

The new recipe files are used to generate test and release versions of the jedi-tutorial singularity container.

## Dependencies

None

## Impact

None

### CODE REVIEW

**Review by rickgrubin — APPROVED** [review]
LGTM

**Review by rachelschweiker — APPROVED** [review]
Nice, Mark!

File: Singularity.tutorial [hunk 1]
```diff
@@ -22,26 +19,32 @@ SPECIES JEDI
...
-    ssh-add /root/.ssh/github_academy_rsa
-    ssh -T -o "StrictHostKeyChecking=no" git@github.com || mkdir -p /opt/jedi
     git config --global credential.helper 'cache --timeout=3600'
     git config --global --add credential.helper 'store'
-    git lfs install
+    git config --global url.ssh://git@github.com/.insteadOf https://github.com/
+    git lfs install --skip-repo
     cd /opt/jedi
-    git clone -b 1.0.0 git@github.com:jcsda/fv3-bundle.git
+    git clone https://github.com/jcsda/fv3-bundle.git
```
Comment (mer-a-o) [inline Singularity.tutorial]: Do we need `-b release/1.1.0` here?

Comment (mmiesch) [reply Singularity.tutorial]: sure - we can do that - done

**Review by mer-a-o — APPROVED** [review]

**Pull request merged**